app/javascripts/components/main.js

```diff
@@ -358,6 +358,7 @@ const GatherPage = React.createClass({
 	},
 
 	toggleCollapseMenu(e) {
+		console.log(this.state)
```

Comment:
could you remove this console.log please?

Reply:
Oops.

Removed.